src/app/core/auth/auth.guard.ts

```diff
@@ -94,15 +94,22 @@ export class AuthGuard implements CanActivate {
 
 			// compile a list of roles that are missing
 			const requiredRoles = route?.data?.roles ?? ['user'];
+			const requireAllRoles = route?.data?.requireAllRoles;
```

Comment:
For backward compatibility, requireAllRoles should default to true if not defined in route data.